reqs/requirements-dev-contrib.txt

```diff
@@ -3,4 +3,4 @@ black[jupyter]==22.3.0
 invoke>=2.0.0
 mypy==1.2.0
 pre-commit>=2.21.0
-ruff==0.0.255
+ruff==0.0.262
```

Comment:
```
contrib/experimental/great_expectations_experimental/expectations/expect_queried_column_value_frequency_to_meet_threshold.py:65:25: C419 [*] Unnecessary list comprehension.
contrib/experimental/great_expectations_experimental/tests/rule_based_profiler/data_assistant/test_growth_numeric_data_assistant.py:288:9: C419 [*] Unnecessary list comprehension.
great_expectations/_version.py:135:13: UP031 [*] Use format specifiers instead of percent format
great_expectations/checkpoint/checkpoint.py:581:17: C419 [*] Unnecessary list comprehension.
great_expectations/checkpoint/types/checkpoint_result.py:81:17: C419 [*] Unnecessary list comprehension.
great_expectations/cli/upgrade_helpers/upgrade_helper_v13.py:162:13: C419 [*] Unnecessary list comprehension.
great_expectations/core/expectation_suite.py:210:13: C419 [*] Unnecessary list comprehension.
great_expectations/core/expectation_validation_result.py:138:21: C419 [*] Unnecessary list comprehension.
great_expectations/core/expectation_validation_result.py:353:21: UP031 [*] Use format specifiers instead of percent format
great_expectations/data_context/types/base.py:1695:21: C419 [*] Unnecessary list comprehension.
great_expectations/data_context/types/base.py:1724:13: C419 [*] Unnecessary list comprehension.
great_expectations/data_context/types/base.py:1752:17: C419 [*] Unnecessary list comprehension.
great_expectations/datasource/data_connector/batch_filter.py:66:20: C419 [*] Unnecessary list comprehension.
great_expectations/datasource/data_connector/configured_asset_sql_data_connector.py:387:21: C419 [*] Unnecessary list comprehension.
great_expectations/datasource/data_connector/file_path_data_connector.py:388:17: C419 [*] Unnecessary list comprehension.
great_expectations/datasource/fluent/pandas_datasource.pyi:56:20: PYI015 [*] Only simple default values allowed for assignments
great_expectations/datasource/fluent/pandas_datasource.pyi:57:21: PYI015 [*] Only simple default values allowed for assignments
great_expectations/datasource/fluent/spark_azure_blob_storage_datasource.pyi:36:60: PYI015 [*] Only simple default values allowed for assignments
great_expectations/datasource/fluent/spark_filesystem_datasource.pyi:30:9: PYI015 [*] Only simple default values allowed for assignments
great_expectations/datasource/fluent/spark_google_cloud_storage_datasource.pyi:33:9: PYI015 [*] Only simple default values allowed for assignments
great_expectations/datasource/fluent/spark_s3_datasource.pyi:30:60: PYI015 [*] Only simple default values allowed for assignments
great_expectations/exceptions/exceptions.py:88:25: C419 [*] Unnecessary list comprehension.
great_expectations/expectations/core/expect_column_quantile_values_to_be_between.py:282:17: C419 [*] Unnecessary list comprehension.
great_expectations/expectations/expectation.py:1431:13: C419 [*] Unnecessary list comprehension.
great_expectations/expectations/expectation.py:1433:24: C419 [*] Unnecessary list comprehension.
great_expectations/expectations/expectation.py:1434:30: C419 [*] Unnecessary list comprehension.
great_expectations/expectations/expectation.py:2554:24: C419 [*] Unnecessary list comprehension.
great_expectations/render/components.py:707:13: C419 [*] Unnecessary list comprehension.
great_expectations/render/components.py:750:13: C419 [*] Unnecessary list comprehension.
great_expectations/rule_based_profiler/data_assistant_result/data_assistant_result.py:3149:20: C419 [*] Unnecessary list comprehension.
great_expectations/rule_based_profiler/data_assistant_result/data_assistant_result.py:3250:20: C419 [*] Unnecessary list comprehension.
great_expectations/rule_based_profiler/data_assistant_result/data_assistant_result.py:4058:20: C419 [*] Unnecessary list comprehension.
great_expectations/rule_based_profiler/domain_builder/categorical_column_domain_builder.py:408:17: C419 [*] Unnecessary list comprehension.
great_expectations/rule_based_profiler/expectation_configuration_builder/default_expectation_configuration_builder.py:277:25: C419 [*] Unnecessary list comprehension.
great_expectations/rule_based_profiler/expectation_configuration_builder/default_expectation_configuration_builder.py:316:30: C419 [*] Unnecessary list comprehension.
great_expectations/rule_based_profiler/helpers/util.py:116:34: C419 [*] Unnecessary list comprehension.
great_expectations/rule_based_profiler/helpers/util.py:160:34: C419 [*] Unnecessary list comprehension.
great_expectations/rule_based_profiler/helpers/util.py:413:13: C419 [*] Unnecessary list comprehension.
great_expectations/rule_based_profiler/helpers/util.py:517:9: C419 [*] Unnecessary list comprehension.
great_expectations/rule_based_profiler/helpers/util.py:548:9: C419 [*] Unnecessary list comprehension.
great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py:753:13: C419 [*] Unnecessary list comprehension.
great_expectations/util.py:1980:44: C419 [*] Unnecessary list comprehension.
great_expectations/validation_operators/types/validation_operator_result.py:55:17: C419 [*] Unnecessary list comprehension.
great_expectations/validation_operators/validation_operators.py:901:17: C419 [*] Unnecessary list comprehension.
tests/actions/test_validation_operators.py:242:9: C419 [*] Unnecessary list comprehension.
tests/actions/test_validation_operators.py:251:9: C419 [*] Unnecessary list comprehension.
tests/actions/test_validation_operators.py:270:9: C419 [*] Unnecessary list comprehension.
tests/actions/test_validation_operators.py:279:9: C419 [*] Unnecessary list comprehension.
tests/actions/test_validation_operators.py:313:9: C419 [*] Unnecessary list comprehension.
tests/actions/test_validation_operators.py:322:9: C419 [*] Unnecessary list comprehension.
tests/actions/test_validation_operators.py:341:9: C419 [*] Unnecessary list comprehension.
tests/actions/test_validation_operators.py:350:9: C419 [*] Unnecessary list comprehension.
tests/conftest.py:365:9: C419 [*] Unnecessary list comprehension.
tests/conftest.py:1072:13: C419 [*] Unnecessary list comprehension.
tests/core/usage_statistics/test_usage_statistics_handler_methods.py:157:16: C419 [*] Unnecessary list comprehension.
tests/core/usage_statistics/util.py:14:9: C419 [*] Unnecessary list comprehension.
tests/core/usage_statistics/util.py:27:9: C419 [*] Unnecessary list comprehension.
tests/datasource/batch_kwarg_generator/test_glob_reader_generator.py:52:16: C419 [*] Unnecessary list comprehension.
tests/datasource/batch_kwarg_generator/test_s3_generator.py:169:16: C419 [*] Unnecessary list comprehension.
tests/datasource/fluent/test_pandas_datasource.py:474:9: C419 [*] Unnecessary list comprehension.
tests/datasource/fluent/test_pandas_filesystem_datasource.py:564:16: C419 [*] Unnecessary list comprehension.
tests/datasource/fluent/test_spark_datasource.py:70:9: C419 [*] Unnecessary list comprehension.
tests/datasource/fluent/test_spark_filesystem_datasource.py:215:16: C419 [*] Unnecessary list comprehension.
Found 63 errors.
```